packages/tre/src/plugins/core/index.ts

```diff
@@ -129,7 +134,10 @@ const LIVE_RELOAD_SCRIPT_TEMPLATE = (
 
 // Using `>=` for version check to handle multiple `setOptions` calls before
 // reload complete.
-export const SCRIPT_ENTRY = `async function handleEvent(event) {
+export const SCRIPT_ENTRY = `
```

Comment:
These scripts are getting a lot more complex. Have you considered different ways of writing them so they're not restricted to string templates? (`fs.readFile` etc...)